tests/invertedIndex.spec.js

```diff
@@ -28,49 +28,49 @@ describe('InvertedIndex Test Suite', () => {
   });
 
   describe('The readFile method', () => {
-    
     it('should read a file', () => {
-      beforeEach(function(){
+      beforeEach(() => {
         const eventListener = jasmine.createSpy();
         var dummyFileReader = { addEventListener: eventListener };
         spyOn(window, "FileReader").and.returnValue(dummyFileReader)
       });
-      var reader = new FileReader();
-      reader.addEventListener('load', function(e) {
-        const fileRead = reader.result
+      const reader = new FileReader();
+      reader.addEventListener('load', (e) => {
+        const fileRead = reader.result;
         expect(fileRead).toBeTruthy();
         expect(JSON.parse(fileRead) instanceof Object).toBeTruthy();
         done();
-     });
+      });
     });
 
     it('should call validateFile', () => {
       var reader = new FileReader();
-      reader.addEventListener('load', function(e) {
+      reader.addEventListener('load', (e) => {
         const fileRead = reader.result
         expect(this.invertedIndex.validateFile(fileRead)).toHaveBeenCalled;
```

Comment:
`this.invertedIndex.validateFile` ? I think if its declared using the static keyword, then you should just access it by the object that created it. You did that on line 17, 89, 94. Why are you instantiating `InvertedIndex` using `this` ? My thoughts sha